centaur/src/main/resources/standardTestCases/delete_intermediates/no_task_no_output_delete.wdl

```diff
@@ -0,0 +1,5 @@
+workflow no_task_no_output_delete {
+  output {
+      Int notUsed = 0
+    }
```

Comment:
ToL: Extra spaces here throwing the indenting off, but I say leave it. IMHO this is the type of thing https://github.com/broadinstitute/winstanley/pull/124 should fix. 😉